src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,68 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.util.Objects;
+
+@Entity
+@Table(name = "users")
```

Comment:
DB tables should be in singular, not plural
```suggestion
@Table(name = "user")
```